### DISCUSSION TITLE
How do I filter all notifications which are unread and for which the underlying PR is not closed?

### DISCUSSION THREAD

**Discussion by smiklosovic** [community/community#77374] [Other Feature Feedback, Questions, & Ideas]
### Notifications

Product Feedback

### Body

There is only `All` or `Unread` button to filter the notifications. What would be great is to filter notifications which are both unread AND for which the PR was not closed. I am not interested in PRs which were closed ... 

There does not seem to be any query which would support this. I tried to do various stuff but to see what I mean (even this query is invalid) it is something like this:

`is:unread and is:open`

![image](https://github.com/community/community/assets/588603/f169bc84-1756-4c7c-8098-169167ad7f4e)

**Comment by 14790897** [comment]
![image](https://github.com/community/community/assets/121866954/d6cfb836-fa7b-42e2-87e2-46f222be7582)

**Reply by smiklosovic** [reply]
That is not the same thing.
